examples/gnn_cost_model/train_cost_model.py

```diff
@@ -0,0 +1,132 @@
+import os
```

Comment:
Could we get a module docstring to explain what this script does? It will be printed if the user runs `python -m gnn_cost_model.train_cost_model --help`